keyboards/telophase/config.h

```diff
@@ -25,13 +25,13 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #define VENDOR_ID       0xFEED
 #define PRODUCT_ID      0x6060
 #define DEVICE_VER      0x0001
-#define MANUFACTURER    unknown
-#define PRODUCT         Telophase
-#define DESCRIPTION     q.m.k. keyboard firmware for Telophase
+#define MANUFACTURER    SouthpawDesign
+#define PRODUCT         telophase
+#define DESCRIPTION     q.m.k. keyboard firmware for TELOPHASE
 
 /* key matrix size */
-#define MATRIX_ROWS 4
-#define MATRIX_COLS 12
+#define MATRIX_ROWS 5
+#define MATRIX_COLS 14
```

Comment:
Is this correct?

Reply:
This comes straight from the manufacturers fork of qmk. It works on my keyboard.

Reply:
According to this: https://southpawdesign.net/products/wireless-split-keyboard-group-buy
the Telophase is actually 4x12, and it's the Helicase that is 5x14.